core/src/main/resources/hudson/PluginManager/index.jelly

```diff
@@ -28,12 +28,7 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <local:table page="updates" list="${app.updateCenter.updates}" xmlns:local="/hudson/PluginManager">
-    <div style="margin-top:1em">
-      <l:isAdmin>
-      ${%Select}: <a href="javascript:toggleCheckboxes(true);">${%All}</a>,
-                  <a href="javascript:checkPluginsWithoutWarnings();">${%Compatible}</a>,
-                  <a href="javascript:toggleCheckboxes(false);">${%None}</a><br/>
-      </l:isAdmin>
+    <div style="margin-left: 0.25rem">
       ${%UpdatePageDescription}
```

Comment:
This should probably be moved to the top of the page, somewhere below the search bar, otherwise it gets lost at the end of the page within the new design.

Reply:
![image](https://user-images.githubusercontent.com/97017043/177185691-ec8ba9f4-1559-428f-b837-a34eb5d95cbc.png)
I have already moved to the top. Please take a look.